### PR TITLE
Feature: Moved sidebar toggle button to address toolbar

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Resolved / Related Issues

*Planned internally*

### Steps used to test these changes

1. Open Files app
2. Resize to be smallest
3. Toggle the button

### CODE REVIEW

File: src/Files.App/UserControls/AddressToolbar.xaml [hunk 1]
```diff
@@ -239,6 +239,28 @@
...
+					x:Name="IsSidebarPaneOpenToggleButton"
+					Width="36"
+					Height="32"
+					Padding="0"
+					x:Load="{x:Bind IsSidebarPaneOpenToggleButtonVisible, Mode=OneWay}"
+					AutomationProperties.Name="ToggleSidebarExpandCollapse"
+					Background="Transparent"
+					BorderBrush="Transparent"
+					IsChecked="{x:Bind MainPageViewModel.IsSidebarPaneOpen, Mode=TwoWay}"
+					ToolTipService.ToolTip="ToggleSidebarExpandCollapse">
```
Comment (yaira2) [inline src/Files.App/UserControls/AddressToolbar.xaml:253]: Please localize the text

File: src/Files.App/UserControls/AddressToolbar.xaml [hunk 2]
```diff
@@ -239,6 +239,28 @@
...
 				Orientation="Horizontal"
 				Spacing="4">
+
+				<ToggleButton
+					x:Name="IsSidebarPaneOpenToggleButton"
+					Width="36"
+					Height="32"
+					Padding="0"
+					x:Load="{x:Bind IsSidebarPaneOpenToggleButtonVisible, Mode=OneWay}"
+					AutomationProperties.Name="ToggleSidebarExpandCollapse"
```
Comment (yaira2) [inline src/Files.App/UserControls/AddressToolbar.xaml:249]:
```suggestion
					AutomationProperties.Name="ToggleSidebarPane"
```

File: src/Files.App/UserControls/AddressToolbar.xaml.cs [hunk 3]
```diff
@@ -15,8 +15,17 @@ namespace Files.App.UserControls
 	public sealed partial class AddressToolbar : UserControl
 	{
 		private readonly IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
+		private readonly MainPageViewModel MainPageViewModel = Ioc.Default.GetRequiredService<MainPageViewModel>();
```
Comment (yaira2) [inline src/Files.App/UserControls/AddressToolbar.xaml.cs:18]: Accessing `MainPageViewModel` from AddressToolbar isn't the best practice, but it'll do for now given the state of the rest of the control.

File: src/Files.App/Views/MainPage.xaml [hunk 4]
```diff
@@ -119,67 +119,50 @@
 
 	<Grid>
 		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
```
Comment (yaira2) [inline src/Files.App/Views/MainPage.xaml:122]: Is this intentional?

File: src/Files.App/Views/MainPage.xaml.cs [hunk 5]
```diff
@@ -108,15 +105,6 @@ private async Task AppRunningAsAdminPromptAsync()
 				UserSettingsService.ApplicationSettingsService.ShowRunningAsAdminPrompt = false;
 		}
 
-		// WINUI3
-		private ContentDialog SetContentDialogRoot(ContentDialog contentDialog)
```
Comment (yaira2) [inline src/Files.App/Views/MainPage.xaml.cs:112]: Is removing this intentional?

File: src/Files.App/UserControls/AddressToolbar.xaml (hunk 1 above)
Comment (0x5bfa) [reply src/Files.App/UserControls/AddressToolbar.xaml]: Done

File: src/Files.App/UserControls/AddressToolbar.xaml (hunk 2 above)
Comment (0x5bfa) [reply src/Files.App/UserControls/AddressToolbar.xaml]: Done

File: src/Files.App/Views/MainPage.xaml (hunk 4 above)
Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml:122]: Yes. this is kind of refactoring; i removed grid that wrapped tab view and address toolbar which is unnecessary.

File: src/Files.App/Views/MainPage.xaml.cs (hunk 5 above)
Comment (0x5bfa) [reply src/Files.App/Views/MainPage.xaml.cs:112]: Yeah not used anywhere

**Pull request closed without merging**